test/pantry-test.js

```diff
@@ -0,0 +1,113 @@
+import {expect} from 'chai';
+import Pantry from '../src/pantry.js';
+import User from '../src/user.js';
+
+describe('Pantry', () => {
+  let testUserData
+  let testIngredientsData
+  let testRecipeData
+
+  beforeEach(() => {
+    testPantry = new Pantry()
+    testUser = new User
+    testUserData = {
```

Comment:
Thanks for creating this `testUserData` object, way better than pulling directly from that large `users.js` in the data directory

Reply:
Thanks! It was some feedback I got for the last project so it was fresh in my head.